modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/query"
 
 	"github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	errors "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
```

Comment:
we typically use `channeltypes` as the alias here, the errors are just in the package along with the other types.

Reply:
Done!